components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml

```
@@ -2971,6 +2982,16 @@ paths:
         Update a Key Manager by keyManager id
       parameters:
       - $ref: '#/components/parameters/keyManagerId'
+      - name: tokenType
+        in: query
+        description: |
+          The type of the tokens to be used (exchanged or without exchanged)
```

```suggestion
          The type of the token to be used (exchanged or without exchanged)
```
This can have only one value ?